src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -319,9 +319,14 @@ export function transform(formConfig, form) {
     ? clonedData.ratedDisabilities.map(d => d.name.toLowerCase())
     : [];
   if (clonedData.newDisabilities) {
-    clonedData.newDisabilities.forEach(d =>
-      claimedConditions.push(d.condition.toLowerCase()),
-    );
+    clonedData.newDisabilities.forEach((d, i) => {
+      const loweredCondition = d.condition.toLowerCase();
+      // PTSD is skipping the cause page and needs to have a default cause of NEW set.
+      if (loweredCondition.includes('ptsd')) {
+        clonedData.newDisabilities[i].cause = 'NEW';
```

Comment:
I think you can do `d.cause = 'NEW';` here and avoid the extra index variable.

Reply:
@jbalboni Linter didn't like mutating the argument "directly". Should I just add an eslint-ignore?

Reply:
Ah, gotcha. Yeah, you can add an eslint ignore comment for that on this line. I think that makes it more obvious what's happening.